packages/util/src/format/formatBalance.ts

```diff
@@ -26,7 +26,7 @@ interface Options {
 }
 
 interface BalanceFormatter {
-  <ExtToBn extends ToBn> (input?: number | string | BN | BigInt | ExtToBn, options?: Options, decimals?: number): string;
+  <ExtToBn extends ToBn> (input?: number | string | BN | BigInt | ExtToBn, options?: Options | boolean, decimals?: number): string;
```

Comment:
This is actually done on purpose - the old-style `boolean` is something that is just kept for historical purposes to not break userspace. (It is deprecated, not yet removed).